src/Fable.Transforms/Replacements.fs

```diff
@@ -2707,17 +2707,40 @@ let asyncs com (ctx: Context) r t (i: CallInfo) (_: Expr option) (args: Expr lis
     // Fable.Core extensions
     | meth -> Helper.CoreCall("Async", Naming.lowerFirst meth, t, args, i.SignatureArgTypes, ?loc=r) |> Some
 
-let guids (_: ICompiler) (ctx: Context) (_: SourceLocation option) t (i: CallInfo) (thisArg: Expr option) (args: Expr list) =
+let guids (com: ICompiler) (ctx: Context) (r: SourceLocation option) t (i: CallInfo) (thisArg: Expr option) (args: Expr list) =
+    let parseGuid (literalGuid: string) =
+        try
+            System.Guid.Parse(literalGuid) |> string |> makeStrConst |> Some
```

Comment:
Maybe the round trip changes the guid format and it's not expected? Should we just use `literalGuid` if parsing is successful?

Reply:
Changing the format of the input to the default format is the default behavior in .NET. 

```fsharp
> Guid("{0x00000000,0x0000,0x0000,{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00}}");;
val it : Guid = 00000000-0000-0000-0000-000000000000

> Guid.Parse("{0x00000000,0x0000,0x0000,{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00}}");;
val it : Guid = 00000000-0000-0000-0000-000000000000

> Guid("{0x00000000,0x0000,0x0000,{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00}}").ToString();;
val it : string = "00000000-0000-0000-0000-000000000000"
```